Package.swift

```diff
@@ -1,32 +1,37 @@
 // swift-tools-version: 5.8
-// The swift-tools-version declares the minimum version of Swift required to build this package.
 // swiftlint:disable explicit_top_level_acl
 // swiftlint:disable prefixed_toplevel_constant
 // swiftlint:disable explicit_acl
 
 import PackageDescription
 
 let package = Package(
-  name: "ContributeWordpress",
+  name: "ContributeWordPress",
+  platforms: [.macOS(.v12)],
```

Comment:
Do we need this?

Reply:
I think we need it, when I removed it I got following error:
```
Compiling for macOS 10.13, but module 'Contribute' has a minimum deployment target of macOS 12.0
```